quickwit/quickwit-opentelemetry/src/otlp/logs.rs

```diff
@@ -47,7 +47,7 @@ use super::{
 use crate::otlp::extract_attributes;
 use crate::otlp::metrics::OTLP_SERVICE_METRICS;
 
-pub const OTEL_LOGS_INDEX_ID: &str = "otel-logs-v0_7";
+pub const OTEL_LOGS_INDEX_ID: &str = "otel-logs-v0_9";
```

Comment:
I would not change the index ID as there is not change in the schema.

Reply:
Yes, I hesitated. My rational was mostly that it would be less confusing for users if logs and traces have the same version number.

Reply:
ok